Content/Items/Misc/Weapons.MagmaGun.Metaballs.cs

```diff
@@ -39,6 +39,12 @@ public override void DrawShapes(SpriteBatch spriteBatch)
 						}
 					}
 				}
+
+				if (proj.active && proj.type == ModContent.ProjectileType<ArrowMagma>())
```

Comment:
Seems a little spaghetti-like to tie up a seperate item into the metaball system for the magma gun, should we consider an alternative design here to keep content coherent together?